app/javascript/controllers/locked_media_poster_controller.js

```diff
@@ -3,16 +3,19 @@ import { Controller } from "@hotwired/stimulus"
 export default class extends Controller {
   // Display the lock if the video tag doesn't have a poster
   show() {
-    if (!this.hasPoster())
-      this.element.style.display = 'flex'
+    if (!this.hasPoster()) {
+      this.element.hidden = false
+      this.element.setAttribute('aria-hidden', 'false');
+    }
   }
 
   hide() {
-    this.element.style.display = 'none'
+    this.element.hidden = true
+    this.element.setAttribute('aria-hidden', 'true');
   }
 
   hasPoster() {
-    const tags = document.querySelectorAll('video,audio')
```

Comment:
Removing the `audio` selector here as a specific audio poster component is added in #1912